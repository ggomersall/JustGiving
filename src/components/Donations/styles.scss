.charity-donations__header {
  display: flex;
}