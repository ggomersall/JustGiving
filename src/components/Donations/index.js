import React from 'react';

import DonationsBio from '../DonationsBio';
import DonationItems from '../DonationItems';
import DonateCarousel from '../DonateCarousel';

import './styles.scss';


const Donations = ({charity, donations}) => {
  return (
    <div className="donations__section container">
      <div className="charity-donations__header">
        <DonationsBio charity={charity} />
        <DonateCarousel charity={charity} />
      </div>
      <DonationItems donations={donations} />
    </div>
  )
}

export default Donations;