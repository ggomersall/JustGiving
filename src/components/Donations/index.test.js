import React from 'react';
import { shallow } from 'enzyme';
import Donations from './index.js';

describe('Testing Donations Component', () => {

  let wrapper;

  test('it should render data correctly', () => {
    const charity = {lorem: ' some text'};
    const donations = {lorem2: ' some text other'};
    wrapper = shallow(<Donations charity={charity} donations={donations} />);
  });

  test('Snapshot', () => expect(<Donations />).toMatchSnapshot());

});

