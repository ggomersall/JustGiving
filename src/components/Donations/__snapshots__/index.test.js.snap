// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Donations Component Snapshot 1`] = `<Donations />`;
